Modules/ITS/src/ITSFhrTask.cxx

```diff
@@ -562,24 +564,36 @@ void ITSFhrTask::monitorData(o2::framework::ProcessingContext& ctx)
         if ((double)GBTLinkInfo->statistics.nTriggers >= 1e6 && (double)GBTLinkInfo->statistics.nTriggers < 1e6 + 10000) {
           mNoisyPixelNumber[mLayer][istave] = 0;
         }
-
         for (int ichip = 0 + (ilink * 3); ichip < (ilink * 3) + 3; ichip++) {
           std::unordered_map<unsigned int, int>::iterator iter;
+
+          if (mDoHitmapFilter == 1 && mTFCount > mCutTFForSparse) {
```

Comment:
We use the same _mHitPixelID_InStave_ vector of digits to draw THnSparse hitmaps. So I decided not to filter vector before the 2D hitmap will not be filled

Reply:
ah I see now! But then you need the `<` sign when filling the sparse. Probably it is still better to separate the variables but I let you think about it.

Reply:
If we have "<" sign there, it would mean that we fill ThnSparse by the same vector each TF until  _mTFCount_ will not became larger _mCutTFForSparse_. I think that it is enough to fill it just once when _mTFCount_ == _mCutTFForSparse_, because we are using function SetBinContent(), and not just a Fill()